src/components/PolicyDetailCard/PolicyDetailCard.tsx

```diff
@@ -0,0 +1,243 @@
+                                <Badge
+                                    variant="lightGreen"
+                                    className="mr-2"
+                                    upperCase={false}
+                                    text="Vigerend (Buiten werking)"
+                                />
+                            ) : (
+                                <Badge
+                                    variant={
+                                        type === 'valid'
+                                            ? 'lightGreen'
+                                            : 'lightRed'
+                                    }
+                                    className="mr-2"
+                                    upperCase={false}
+                                    text={policy.Status || ''}
+                                />
+                            )}
+                            {policyIsInAModule
+                                ? policy.Ref_Beleidsmodules?.map(module => (
+                                      <Badge
+                                          key={module.UUID}
+                                          variant={'gray'}
+                                          upperCase={false}
+                                          text={module.Titel || ''}
+                                          className="mr-2"
+                                      />
+                                  ))
+                                : null}
+                        </div>
+                    </div>
+                    <div className="mt-2">
+                        {type === 'checked out' ? (
+                            <Hyperlink
+                                to={`/muteer/${overviewSlug}/edit/${policy.ID}`}
+                                text="Bewerken"
+                                icon={ArrowUpRightFromSquare}
+                            />
+                        ) : null}
+                        {type === 'valid' ? (
+                            <Hyperlink
+                                to={`/detail/${overviewSlug}/${policy.UUID}`}
+                                text={`Bekijk ${titleSingular.toLowerCase()}`}
+                                icon={ArrowUpRightFromSquare}
+                            />
+                        ) : null}
+                    </div>
+                </div>
+            </Card>
+        </motion.div>
+    )
+}
+
+type LeftLineIndicatorProps = {
+    type: PossibleCardTypes
+    lastItem: boolean
+}
+
+const LeftLineIndicator = ({
+    type,
+    lastItem = false,
+}: LeftLineIndicatorProps) => {
+    if (type === 'checked out') {
+        return (
+            <div>
+                <ColoredBall
+                    color="red"
+                    className="top-[5px]"
+                    style={{
+                        marginLeft: 'calc(1.5rem - 4px)',
+                    }}
+                    animated={true}
```

Comment:
@Aidenbuis Another small thing; if the value is true by default you can just add the `animated` prop without any value.